plugins/embed-optimizer/tests/test-cases/all-embeds-inside-viewport/expected.html

```diff
@@ -2,6 +2,8 @@
 	<head>
 		<meta charset="utf-8">
 		<title>...</title>
+		<meta data-od-replaced-content="optimization-detective 0.0.0" name="generator" content="optimization-detective 0.0.0; url_metric_groups={0: complete, 480: complete, 600: complete, 782: complete}">
```

Comment:
Note the `data-od-replaced-content` attribute is not ideal. This will have to be addressed in a separate issue, however: https://github.com/WordPress/performance/issues/1931#issuecomment-2755179868